wix/Includes/OpenJDK.Variables.wxi

```diff
@@ -11,13 +11,13 @@
     <?define Arch="(x64)" ?>
   <?endif?>
 
-  <?define ProductVersion="$(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion)" ?>
+  <?define ProductVersion="$(var.MSIProductVersion)" ?>
   <?define ProductLanguage="!(loc.ProductLanguage)" ?>
   <?define ProductManufacturer="AdoptOpenJDK" ?>
 
   <?if $(env.PRODUCT_CATEGORY)="jre"?>
     <?define ProductName="!(loc.JREProductName)" ?>
-    <?define ProductNameWithVersion="!(loc.JREProductName) $(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion) $(var.Arch)" ?>
+    <?define ProductNameWithVersion="!(loc.JREProductName) $(var.MSIProductVersion) $(var.Arch)" ?>
```

Comment:
Sxa and I think that it is better to show full version info here and not computed one.
11.0.9.1+1 is better than
11.0.9.101

Reply:
Sure. Have updated it.